app/components/works/contributor_row_component.rb

```diff
@@ -4,14 +4,26 @@
 module Works
   # Renders a widget corresponding to a single contributor to the work.
   class ContributorRowComponent < ApplicationComponent
-    sig { params(form: ActionView::Helpers::FormBuilder).void }
-    def initialize(form:)
+    sig { params(form: ActionView::Helpers::FormBuilder, required: T::Boolean, citation: T::Boolean).void }
```

Comment:
Is it worth considering splitting this component into two components: the current ContributorRowComponent and a new AuthorRowComponent? Or do you think that wouldn't get us much of value? I'm assuming the added complexity in this component is driven by using it in both contexts, but I may be off.

Reply:
I don't fee this is too complex. It's effectively one boolean (as required and citation are always set the same), only more expressive.